go/grpcwebproxy/backend.go

```diff
@@ -30,6 +30,12 @@ var (
 		"Whether to ignore TLS verification checks (cert validity, hostname). *DO NOT USE IN PRODUCTION*.",
 	)
 
+	flagMaxMsgSize = pflag.Int(
+		"backend_max_msg_size",
+		4194304,
```

Comment:
Can we make this equivalent to the current max? https://github.com/grpc/grpc-go/blob/v1.8.2/server.go#L54. Also please add a reference to this line on github (i.e., copy this link). Note that the link is for v1.8.2 which is the version we've locked in Gopkg.toml.

Reply:
I thought the max is 4MB, I'll update